### DISCUSSION TITLE
Adding a multiaccounting warning when logging into osu!lazer

### DISCUSSION THREAD

**Discussion by raybeanosu** [ppy/osu#15606] [Ideas] [answered]
This might not be something that is 100% required, as not everyone might use lazer to evade multiaccounting, as scores don't get submitted _with pp._ However, having one similar to that seen in osu! stable would be a good idea.

Here's an idea as to how I think it can look like. It's not perfect as I'm not that great at design, but it should suffice for now hopefully. 

![image](https://user-images.githubusercontent.com/91675300/141645140-1ed2165f-a8b4-439f-8e71-706d9a475b30.png)

**Comment by bdach** [comment, marked as answer]
this already exists.

![2021-11-13-142527_820x613_scrot](https://user-images.githubusercontent.com/20418176/141645493-aa3aecfd-dcd4-4997-85a0-7c0d59180f20.png)

**Reply by raybeanosu** [reply]
oh okay, thanks for informing me! this wasn't there when I had last seen it.

**Reply by peppy** [reply]
It's been there from day one. It will only show when it is required, though.

**Reply by raybeanosu** [reply]
I see how it is, thank you peppy! wasn't aware of it.
